student.rb

```diff
@@ -2,7 +2,7 @@
 
 class Student < Person
   def initialize(age, parent_permission, classroom)
-    super(name, age, parent_permission)
+    super(age, parent_permission)
     @classroom = classroom
   end
```

Comment:
In the **Student** class, you need to have a method that serves as the `belongs-to` side (a student belongs to a classroom). Kindly create a separate method `def classroom=(classroom)`. In that method you should establish belongs to a relationship with the student.